taxi/views.py

```diff
@@ -87,3 +112,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+    @staticmethod
+    def post(request, *args, **kwargs):
+        driver = get_object_or_404(Driver, pk=kwargs["pk"])
+        form = DriverLicenseUpdateForm(request.POST)
```

Comment:
In this case to use form.save()
Create form as:
form = DriverLicenseUpdateForm(request.POST, instance=driver)